io.catenax.time_series_reference/1.0.0/TimeSeriesReference.ttl

```diff
@@ -0,0 +1,156 @@
+   bamm:characteristic :DelimiterChoice ;
+   bamm:exampleValue "semicolon" ;
+   bamm:name "delimiter" .
+
+:payload a bamm:Property ;
+   bamm:preferredName "payload"@en ;
+   bamm:description "contains the information about units and variable names"@en ;
+   bamm:characteristic :PayloadCharacteristic ;
+   bamm:name "payload" .
+
+:DecimalSeperatorChoice a bamm-c:Enumeration ;
+   bamm:dataType xsd:string ;
+   bamm-c:values ( "dot" "comma" ) ;
+   bamm:name "DecimalSeperatorChoice" .
+
+:ResourcePath a bamm:Characteristic ;
+   bamm:preferredName "Resource Path"@en ;
+   bamm:description "The path of a resource."@en ;
+   bamm:dataType xsd:anyURI ;
+   bamm:name "ResourcePath" .
+
+:FileExtensionChoice a bamm-c:Enumeration ;
+   bamm:dataType xsd:string ;
+   bamm-c:values ( "csv" "none" "txt" "dat" "data" ) ;
+   bamm:name "FileExtensionChoice" .
+
+:DelimiterChoice a bamm-c:Enumeration ;
+   bamm:dataType xsd:string ;
+   bamm-c:values ( "semicolon" "comma" "tab" ) ;
+   bamm:name "DelimiterChoice" .
+
+:PayloadCharacteristic a bamm-c:TimeSeries ;
+   bamm:preferredName "payload characteristic"@en ;
+   bamm:dataType :PayloadEntity ;
+   bamm:name "PayloadCharacteristic" .
+
+:PayloadEntity a bamm:Entity ;
+   bamm:preferredName "payload entity"@en ;
+   bamm:description "header payload"@en ;
+   bamm:properties ( :units :variableNames :timeColumnName ) ;
+   bamm:name "PayloadEntity" .
+
+:units a bamm:Property ;
+   bamm:description "lists the unit for each datapoint"@en ;
+   bamm:characteristic :UnitsListCharacteristic ;
+   bamm:name "units" .
+
+:variableNames a bamm:Property ;
+   bamm:description "lists the name for each datapoint"@en ;
+   bamm:characteristic :VariableListCharacteristic ;
+   bamm:name "variableNames" .
+
+:timeColumnName a bamm:Property ;
+   bamm:preferredName "time column"@en ;
+   bamm:description "specifies the name of the variableNames entry which contains the time column"@en ;
+   bamm:characteristic :TimeColumnChoice ;
+   bamm:name "timeColumnName" .
+
+:UnitsListCharacteristic a bamm-c:List ;
+   bamm:description "lists the unit for each datapoint"@en ;
+   bamm:dataType :Unit ;
+   bamm:name "UnitsListCharacteristic" .
+
+:VariableListCharacteristic a bamm-c:List ;
+   bamm:description "lists the name for each datapoint"@en ;
+   bamm:dataType :VariableName ;
+   bamm:name "VariableListCharacteristic" .
+
+:TimeColumnChoice a bamm-c:Enumeration ;
+   bamm:preferredName "time column name"@en ;
+   bamm:description "specifies the name of the time colums variable"@en ;
+   bamm:dataType xsd:string ;
+   bamm-c:values ( "Time" "TIME" "time" "t" ) ;
+   bamm:name "TimeColumnChoice" .
+
+:Unit a bamm:Entity ;
+   bamm:description "unit of a column"@en ;
+   bamm:properties ( :unit ) ;
+   bamm:name "Unit" .
+
+:VariableName a bamm:Entity ;
+   bamm:description "name of a column "@en ;
+   bamm:properties ( :variableName ) ;
```

Comment:
What does this entity bring into the model if it only has one property? Wouldn't it be sufficient to use the variable name property directly? ie using string as datatype in variableListCharacteristic?

Reply:
Yes you are right